File: parsers/python-zipfile/unzip.py
from zipfile import ZipFile
from sys import argv

zip = ZipFile(argv[1], 'r')
error_file = zip.testzip()
if error_file is None:
    zip.extractall(argv[2])
else:
    print(f"Error in file {error_file}")
    exit(1)

File: parsers/csharp-dotnetzip/unzip.cs
using Ionic.Zip;

if (ZipFile.CheckZip(args[0]))
{
    using (ZipFile zip = ZipFile.Read(args[0]))
    {
        zip.ExtractAll(args[1], ExtractExistingFileAction.OverwriteSilently);
    }
}
else
{
    System.Environment.Exit(1);
}
